UI.md

```diff
@@ -2,7 +2,7 @@
 
 1. Avoid put more than 5 React components in a single file. Use separate files for big components or components that are used in multiple places.
 2. Avoid files with only a few lines of code that are overly simple, rather try group them meaningfully.
-3. Put multiline styles in the top of the file.
+3. Put multiline styles in the top of the file. [not sure i agree with this, inline is easier unless you're repeating the style]
 4. Prefer template strings over string concatenation.
 5. Use SWR if fetching data (outside of the graph) that is useful to persist between pages.
-6. Use react context for state data between pages.
+6. Use react context for state data between pages. [Would appreciate some more info as to when to choose between SWR and react context / providers.]
```

Comment:
Ok, "state data" is things like, is the user logged in or which network they are on or things lilke that.

For other things that are just fetching data from somewhere, I think SWR is better. It is easier to think about, and you don't have to have any complexity related to contexts etc. So I think currently we are over-using the context.

Will write more about this.